setup.cfg

```
@@ -10,25 +10,24 @@ url = https://github.com/stactools-naips/stactools-naip
 project_urls =
     Documentation = https://stactools-naip.readthedocs.io/en/latest/
     Issues = https://github.com/stactools-naips/stactools-naip/issues
-keywords = 
+keywords =
     stactools
     pystac
     catalog
     STAC
 classifiers =
     Development Status :: 4 - Beta
     License :: OSI Approved :: Apache Software License
-    Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
 
 [options]
 package_dir =
-    = src
+    =src
 packages = find_namespace:
 install_requires =
-    stactools == 0.2.1a1
+    stactools ~= 0.3.0
```

```suggestion
    stactools >= 0.3.0
```

Obviously dependency specification is squirrely, but I _think_ I'm convinced that `>=` is better than `~=` for Python (written up https://www.gadom.ski/2022/02/18/dependency-protection-with-python-and-github-actions.html).